packaging/rpm/make-rpm.sh

```diff
@@ -76,10 +101,24 @@ usage() {
 [ $# -lt 2 ] && usage
 
 case $1 in
-  all)  RPMBUILD_OPT=-ba ;;
-  rpm)  RPMBUILD_OPT=-bb ;;
-  srpm) RPMBUILD_OPT=-bs ;;
-  *)    usage
+  all)
+    RPMBUILD_OPT=-ba
+    RPM_INFO_DIRS="RPMS SRPMS"
```

Comment:
`RPM_INFO_DIRS` does not seem to be used.

Reply:
It is used in print_info function, but it should be an argument instead.

Reply:
OK, let's fix it for consistency with other funcs.